wapitiCore/net/web.py

```diff
@@ -479,6 +479,10 @@ def file_path(self):
     def is_root(self) -> bool:
         return self._file_path == "/"
 
+    @property
+    def root(self) -> str:
```

Comment:
Maybe use something less ambigous than `root` ?

Reply:
`website` ? `webroot` ? :thinking:

Reply:
Oh i think i misunderstood the behavior of urljoin with `/`